source/linux/Foundational_Components/Kernel/Kernel_Drivers/Storage/MMC-SD.rst

```diff
@@ -294,160 +302,231 @@ Driver Configuration
    .. rubric:: **Enabling eMMC Card Background operations support**
       :name: enabling-emmc-card-background-operations-support
 
-   | eMMC cards need to occasionally spend some time cleaning up garbage
-     and perform cache/buffer related operations which are strictly on the
-     card side and do not involve the host. These operations are at various
-     levels based on the importance/severity of the operation 1- Normal, 2-
-     Important and 3 - Critical. If an operation is delayed for long it
-     becomes critical and the regular read/write from host can be delayed
-     or take more time than expected.
-   | To avoid such issues the MMC HW and core driver provide a framework
-     which can check for pending background operations and give the card
-     some time to clear up the same.
-   | This feature is already part of the framework and to start using it
-     the User needs to enable EXT\_CSD : BKOPS\_EN [163] BIT 0.
+   eMMC cards need to occasionally spend some time cleaning up garbage
+   and perform cache/buffer related operations which are strictly on the
+   card side and do not involve the host. These operations are at various
+   levels based on the importance/severity of the operation 1- Normal, 2-
+   Important and 3 - Critical. If an operation is delayed for long it
+   becomes critical and the regular read/write from host can be delayed
+   or take more time than expected.
+   To avoid such issues the MMC HW and core driver provide a framework
+   which can check for pending background operations and give the card
+   some time to clear up the same.
+   This feature is already part of the framework and to start using it
+   the User needs to enable EXT\_CSD : BKOPS\_EN [163] BIT 0.
 
    **This can be done using the "mmc-utils" tool from user space or using
    the "mmc" command in U-boot.**
 
    Command to enable bkops from userspace using mmc-utils, assuming eMMC
    instance to be mmcblk0
 
-   ::
+   .. code-block:: console
 
-       root@dra7xx-evm:mmc bkops enable /dev/mmcblk0
+      root@dra7xx-evm:mmc bkops enable /dev/mmcblk0
```

Comment:
```suggestion
      root@<machine>:mmc bkops enable /dev/mmcblk0
```

Since it's under `.. ifconfig:: CONFIG_part_family in ('General_family', 'AM335X_family', 'AM437X_family')`